src/Symfony/Component/Runtime/Runner/Symfony/HttpKernelRunner.php

```diff
@@ -31,7 +33,21 @@ public function __construct(
     public function run(): int
     {
         $response = $this->kernel->handle($this->request);
-        $response->send(!$this->debug);
+
+        if (Kernel::VERSION_ID >= 60400) {
+            $response->send(false);
+
+            if (\function_exists('fastcgi_finish_request') && !$this->debug) {
+                fastcgi_finish_request();
+            } elseif (\function_exists('litespeed_finish_request') && !$this->debug) {
+                litespeed_finish_request();
+            } else {
+                Response::closeOutputBuffers(0, true);
```

Comment:
Unlike what the logic in `$response->send()`, we call this unconditionally from the SAPI. The reason is that here, in the runtime, we know we want to flush.